### ISSUE TITLE
appservice user should not be given Admin to the plumbed room by default

### ISSUE THREAD

**Issue by alaviss** [matrix-org/matrix-appservice-irc#1396]
**Describe the bug**
If you plumb a room, the `appservice` user will be invited and be given "Admin". This makes it hard for the human admins to kick the bridge in case it malfunctions and doesn't quit.

**To Reproduce**
1. Create a new room
2. Plumb it to libera.chat

**Expected behavior**
The bridge should be at most given a PL above Moderator (ie. 51). It should have enough PL to function in the plumbed room in that case.

**Comment by namazso** [comment]
consider creating your rooms with a higher-than-admin PL, solves lots of these issues

**Comment by Mikaela** [comment]
> consider creating your rooms with a higher-than-admin PL, solves lots of these issues

Are the steps to do this actually documented anywhere for average user to understand? Is there an issue to Element (Web, iOS and Android) for doing that? How about other clients?

**Comment by namazso** [comment]
> Are the steps to do this actually documented anywhere

Yes: https://matrix.org/docs/spec/client_server/latest#post-matrix-client-r0-createroom

> for average user to understand? 

No.


Use postman / whatever your preferred tool is and send off the request manually. Obviously put your bearer token in too.

![image](https://user-images.githubusercontent.com/8676443/125168866-d4bee380-e1a7-11eb-9d69-f90846005aa4.png)
